### ISSUE TITLE
[$150]THRIVE - Practical Skills Needed for Starting Data Science - In Layman Terms!

### ISSUE THREAD

**Issue by rj97** [topcoder-platform/thrive-submit#30]
Track - Data Science
[Article 1 - Practical skills needed to win data science competitions](https://github.com/topcoder-platform/thrive-submit/files/3928317/Article.1.-.Practical.skills.needed.to.win.data.science.competitions.docx)

**Comment by kkartunov** [comment]
passed Pcheck

**Comment by harshitmehta123** [comment]
Great Article @rj97. How about changing the title to something more attractive to the new audience. 

**Practical Skills Needed To For Data Science - In Layman Terms!**

**Comment by rj97** [comment]
> **Practical Skills Needed for Starting Data Science - In Layman Terms!**

@harshitmehta123 sounds good! Thanks for reviewing the article.

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30110978 has been created for this ticket.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
@rj97, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30110978 has been updated - rjparmar has been unassigned.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```
